usaspending_api/api_contracts/contracts/v2/download/count.md

```diff
@@ -38,17 +43,26 @@ Returns the number of transactions that would be included in a download request
                     "recipient_type_names": ["higher_education"],
                     "place_of_performance_locations": [{"country": "USA", "state": "WI"}],
                     "psc_codes": {"require": [["Product","10","1035"]]}
-                }
+                },
+                "spending_level": "awards"
             }
 
 + Response 200 (application/json)
     + Attributes (object)
-        + `transaction_rows_gt_limit` (required, boolean)
-            A boolean returning whether the transaction count is over the maximum row limit.
         + `calculated_transaction_count` (required, number)
             The calculated count of all transactions which would be included in the download files.
         + `maximum_transaction_limit` (required, number)
             The current allowed maximum number of transactions in a row-limited download. Visit https://www.usaspending.gov/download_center/custom_award_data to download larger volumes of data.
+        + `transaction_rows_gt_limit` (required, boolean)
```

Comment:
Don't forget to put these in the sample body response down below with sample data as well